kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/ClustersProperties.java

```diff
@@ -62,6 +63,14 @@ public static class SchemaRegistryAuth {
     String password;
   }
 
+  @Data
+  public static class SchemaRegistrySsl {
```

Comment:
Lets generalise it a little for possible further reuse -> pls rename class to `WebClientSsl`

Reply:
renamed in commit c86b2286557138fdc6c5473a1f35baa5ef75043f. Additionally adapted the field naming in `InternalSchemaRegistry`